app/renderer/src/reducers/copiedToClipboard.js

```diff
@@ -0,0 +1,12 @@
+import {
+  COPY_ITEM,
+} from '../actions/types';
+
+export default function (state, { type }) {
```

Comment:
@vutran Actually, I thought about it and I figured that the only action that should set the flag to true is `COPY_ITEM`, so every other actions should turn it off. I think this makes sense?